allocator.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+#
+# Share NVIDIA_VISIBLE_DEVICES to ConfigMap
+
+from kubernetes import client, config
+from kubernetes.client.rest import ApiException
+import os
+
+# Read k8s config from environment KUBERNETES environment variables
+config.load_incluster_config()
+
+api_instance = client.CoreV1Api()
+name = os.environ.get("GPU_CONFIGMAP", "gpu-config")
```

Comment:
From my reading of the other scripts, we shouldn't be executing allocator unless this is set. So it could make sense to asset it's present rather than allowing a default here?